calculation/sha_calc/sha_calc/directivity/bea20/utils.py

```diff
@@ -67,11 +67,8 @@ def set_hypocentres(n_hypo: int, planes: List, depth_method: List):
         current_length = 0
         planes_copy = [plane.copy() for plane in planes]
         for index, plane in enumerate(planes_copy):
-            if (
-                current_length < distance
-                and (current_length + plane["length"]) > distance
-            ):
-                plane["shyp"] = distance - (current_length + plane["length"] / 2)
+            if current_length < distance < (current_length + plane["length"]):
+                plane["shyp"] = distance - (total_length / 2)
```

Comment:
Just checking, line 70 is just a simplified version
but line 71 changed from 
```python
plane["shyp"] = distance - (current_length + plane["length"] / 2)
```
to 
```python
plane["shyp"] = distance - (total_length / 2)
```
Is this meant to be happened?

Reply:
Yes this was meant to happen, the shyp value was meant to be from the centre of the plane it was in, but turns out it was actually from the centre of the entire fault and so this adjustment was needed so that GC2 gets the correct location for the hypocentre.